index/heidegger-index.yml

```diff
@@ -548,11 +552,15 @@ Dionysos:
     - start: 271
   type: p
 Dostojevski:
+  metadata:
+    gnd_id: '118527053'
```

Comment:
Hier is het een `str` en op andere plekken een `int`. Zouden we niet een uniform type willen hebben?

Reply:
De GND-ID bestaat uit 9 of 10 nummers met eventueel een 'X' als het 'controlegetal' 10 zou moeten zijn. Hierom moet het dus in principe een `str` zijn, en geen `int` als we consequent willen zijn. Ik zal dit aanpassen in de validatie en de invoer.